transactions/src/main/java/org/jboss/as/txn/subsystem/TransactionSubsystemAdd.java

```diff
@@ -137,6 +137,14 @@ private void populateModelWithObjectStoreConfig(ModelNode operation, ModelNode o
         TransactionSubsystemRootResourceDefinition.OBJECT_STORE_RELATIVE_TO.validateAndSet(operation, objectStoreModel);
         TransactionSubsystemRootResourceDefinition.OBJECT_STORE_PATH.validateAndSet(operation, objectStoreModel);
 
+        ModelNode relativeVal = objectStoreModel.get(TransactionSubsystemRootResourceDefinition.OBJECT_STORE_RELATIVE_TO.getName());
+        ModelNode pathVal =  objectStoreModel.get(TransactionSubsystemRootResourceDefinition.OBJECT_STORE_PATH.getName());
+
+        if (!relativeVal.isDefined() &&
+                (!pathVal.isDefined() || pathVal.asString().equals(TransactionSubsystemRootResourceDefinition.OBJECT_STORE_PATH.getDefaultValue().asString()))) {
```

Comment:
@zhfeng Yes, we should do this. The parser change ensures that old config files produce the same results they did before; this change helps ensure that old CLI scripts produce the same results they did before.